tests/test_browser.py

```diff
@@ -5121,6 +5121,22 @@ def test_zzz_zzz_4gb_fail(self):
     self.emcc_args += ['-O2', '-s', 'ALLOW_MEMORY_GROWTH', '-s', 'MAXIMUM_MEMORY=4GB', '-s', 'ABORTING_MALLOC=0']
     self.do_run_in_out_file_test('browser', 'test_4GB_fail.cpp')
 
+  # Tests that Emscripten-compiled applications can be run when a slash in the URL query or fragment of the js file
+  def test_browser_run_with_slash_in_query_and_hash(self):
+    src = open(path_from_root('tests', 'browser_test_hello_world.c')).read()
+    create_file('test.c', src)
+    self.compile_btest(['test.c', '-o', 'test.html', '-O3'])
```

Comment:
You can just do `self.compile_btest([test_file('tests/browser_test_hello_world.c'), '-o', 'test.html', '-O3'])` here.. no need to create a new local source file.

Reply:
Any reason for `-O3` here?    Leaving it as the default (`-O0`) can make things easier to debug.